plugins/SitesManager/vue/src/SiteWithoutData/SiteWithoutData.vue

```diff
@@ -56,20 +48,38 @@ export default defineComponent({
       type: String,
       required: true,
     },
+    descriptionLine1AdditionalLinks: {
+      type: String,
+      required: true,
+    },
     afterIntroEventContent: String,
     afterTrackingHelpEventContent: String,
   },
   components: {
-    ContentBlock,
     WidgetLoader,
     VueEntryContainer,
   },
   computed: {
-    siteWithoutDataDesc() {
+    siteWithoutDataDescLine1() {
       return translate(
-        'SitesManager_SiteWithoutDataDescription',
+        'SitesManager_SiteWithoutDataStartTrackingDataDescriptionLine1',
         `<a href="${this.emailInstructionsLink}">`,
         '</a>',
+        this.descriptionLine1AdditionalLinks,
```

Comment:
We should maybe think about another solution for this. Concatenating and inserting translations into other translations makes it very very hard for the translators to translate that properly. In some cases that even makes it impossible in terms of grammar. We should use placeholders only to either insert a certain value/number or html tags.

Wouldn't it work to simply inject the key of the translation to use instead of the additional links variable. We could in the cloud plugin then inject a translation that already contains an additional link or similar.

Reply:
Added more cleaner approach as discussed over slack, thanks for the help @sgiehl :smile: